src/main/scala/cromwell/Main.scala

```diff
@@ -333,18 +341,25 @@ class Main private[cromwell](enableTermination: Boolean, managerSystem: () => Wo
     a <- Actions.values find (_.toString == argCapitalized)
   } yield a
 
-  private[this] def loadWdl(path: String)(f: WdlNamespace => Int): Int = {
-    Try(WdlNamespace.load(new JFile(path), WorkflowManagerActor.BackendType)) match {
-      case Success(namespace) => f(namespace)
-      case Failure(t) =>
-        println(t.getMessage)
-        exit(1)
+  private[this] def loadWdl(args: Seq[String])(f: WdlNamespace => Int): Int = {
+    /*
+      As of this writing all functions which call this have >= 2 args and are such that args(0) is the path
+      and args(1) is the backend type. Those functions will be going away soon so it's unlikely that something
+      runs afoul of this 
```

Comment:
All of the Main endpoints except for server and run are being moved out of the cromwell jar as part of the WDL/Cromwell decoupling: DSDEEPB-2301. Because of this I don't really care if the parse/highlight/inputs/etc are a little janky for the moment as they'll be gone in a week or three and when they reappear they won't have knowledge of backend.

Backend is currently being massively refactored by our chip making friends and we'll find out exactly how some point next month

Neither have any bearing on a release IMO